src/map/conquest_system.h

```diff
@@ -60,8 +50,9 @@ namespace conquest
     uint8 GetInfluenceRanking(int32 san_inf, int32 bas_inf, int32 win_inf, int32 bst_inf);
     uint8 GetInfluenceRanking(int32 san_inf, int32 bas_inf, int32 win_inf);
 
-    void UpdateConquestGM(ConquestUpdate type); // Update conquest system by GM (modify in the DB and use @updateconquest)
-    void UpdateWeekConquest();                  // Update conquest system every sunday
+    void HandleInfluenceUpdate(const std::vector<influence_t>& influences, bool shouldUpdateZones); // Triggered periodically when world server sends updates of the latest influence data.
```

Comment:
We're an east const (EAST IS BEAST) family around here ;)

(apart from const char*, because what is life without nonsensical conventions)

Reply:
Done